auth_proxy/models/user.py

```diff
@@ -16,3 +17,26 @@ class User(Base):
 
     id = Column(Integer, primary_key=True)
     patient_id = Column(String)
+    username = Column(String)
+    password = Column(String)
```

Comment:
This is a little silly, but I'm still weirded out seeing even a fixturized password stored plain in a db. Would you object to hashing (e.g. with https://pypi.python.org/pypi/bcrypt/3.1.0)?